set.mm

```diff
@@ -300838,22 +301071,19 @@ unit group (that is, the nonzero numbers) to the field.  (Contributed
       EUHUIUKUBUJCUCUD $.
   $}
 
-  ${
-    distspace.x $e |- X = ( Base ` G ) $.
-    distspace.d $e |- D = ( dist ` G ) $.
-    $( A structure ` G ` with a distance function ` D ` which is a pseudometric
-       is a _distance space_ (according to E. Deza, M.M. Deza:  "Dictionary of
-       Distances", Elsevier, 2006), i.e. a (base) set ` X ` equipped with a
-       _distance_ ` D ` , which is a mapping of two elements of the base set to
-       the (extended) reals and which is non-negative, symmetric and equal to 0
-       if the two elements are equal.  (Contributed by AV, 15-Oct-2021.) $)
-    distspace $p |- ( ( D e. ( PsMet ` X ) /\ A e. X /\ B e. X )
-                      -> ( ( D : ( X X. X ) --> RR* /\ ( A D A ) = 0 )
-                          /\ ( 0 <_ ( A D B ) /\ ( A D B ) = ( B D A ) ) ) ) $=
-      ( cpsmet cfv wcel w3a cxp cxr wf co cc0 wceq wa cle wbr psmetf psmet0 jca
-      3ad2ant1 3adant3 psmetge0 psmetsym jca32 ) CEHIJZAEJZBEJZKZEELMCNZAACOPQZ
-      RPABCOZSTUOBACOQULUMUNUIUJUMUKCEUAUDUIUJUNUKACEUBUEUCABCEUFABCEUGUH $.
-  $}
+  $( A set ` X ` together with a (distance) function ` D ` which is a
+     pseudometric is a _distance space_ (according to E. Deza, M.M. Deza:
+     "Dictionary of Distances", Elsevier, 2006), i.e. a (base) set ` X `
+     equipped with a _distance_ ` D ` , which is a mapping of two elements of
+     the base set to the (extended) reals and which is non-negative, symmetric
+     and equal to 0 if the two elements are equal.  (Contributed by AV,
```

Comment:
That's an extended pseudometric space. Why invent new names ? "Distance space" has a much more ambiguous meaning. This removes the need for the reference.

Reply:
I did not invent this name, but took it from the referenced book. If this book is not serious, I will remove the reference, otherwise I would like to keep it. Is there a reference where the term "extended pseudometric space" is defined? This could be added here.

Reply:
> > With regards to your question on a previous PR, in this example, there could have been one commit for each bullet point.
> 
> This would impede my personal workflow, because I work in a separate working directory (using some temporary files, and to avoid unintended changes in "official" files). To synchronize the changes with the git directory more often would cause some overhead and wold increase the risk of "distroying" something.

Furthermore, I did not perform the chages in a sequence: the main task here was the removal of the superfluous hypotheses, the other changes are made "en passant".

Reply:
In short: I'm going to open an Issue (see below), but for now, leave it like that.

As for "extended pseudometric": I don't know for the combination, but:
- "extended metric" is standard for "metric with values in the (nonnegative) extended reals",
- "pseudometric" is standard for "nonseparated metric".

For these, see https://en.wikipedia.org/wiki/Metric_(mathematics).

From these two, "extended pseudometric" is self-explanatory.  Bourbaki calls that notion "écart", I don't know how it is translated in English (in common language, "écart" means "spacing" or "gap"), but I think a new word is not needed.

Note: "quasimetric" is standard (maybe a bit less than the other two above) for "nonsymmetric metric".  So one could introduce a notion of "extended quasipseudometric".  Even though this may look like "centipede mathematics" (*), this is actually a natural notion, since it corresponds to $([0,+\infty], \leq, +)$-enriched categories, as noted by Lawvere.  Hence, the "crucial" property of metrics is the triangle inequality, more than the other axioms.

As for the seriousness of the reference, it would be out of place for me to judge it.  What I can say is that in many specialized handbooks or dictionaries, one is more likely to find specialized terms used only within a smaller community, that may have diverged overtime. (And indeed no one is immune to this kind of bias; @digama0 recently remarked here that I may bend too much toward category theory language, which is less universal than I thought/hoped.)

I just had a look at df-psmet and nearby definitions df-xmet and df-met... We should rewrite these definitions, with each in terms of the "previous" (like magma > semigroup > monoid > group). I'm going to open an Issue.

(*) "centipede mathematics" (I think the term was coined by John Baez) is the tendency to study generalizations of usual notions without any motivation, hence removing axioms one by one, like the legs of a centipede, and seeing what properties remain. A symptom is the multiplication of terms like pseudo-smurf, quasi-, almost-, near-, semihemi-... Sometimes it is justified, but the justification should not be forgotten.

Reply:
Yes, it is a good idea to discuss this separately, because it is off topic here (this PR is about removal of hypotheses).